alphacsc/_d_solver.py

```diff
@@ -1,22 +1,84 @@
 import numpy as np
 
-from .init_dict import init_dictionary
+from .init_dict import get_dict_strategy
```

Comment:
I would use `init_strategy` instead of `dict_strategy`.

Reply:
Changed.